core/textile/bucket_factory.go

```diff
@@ -408,13 +409,20 @@ func (tc *textileClient) ToggleBucketBackup(ctx context.Context, bucketSlug stri
 
 	if bucketSchema.Backup {
 		tc.sync.NotifyBucketBackupOn(bucketSlug)
+		tc.sync.NotifyBucketRestore(bucketSlug)
```

Comment:
Not fully clear on why we need to start a restore when backup is just toggled on?

Reply:
@jsonsivar 

Thank you for spotting this. Removed https://github.com/FleekHQ/space-daemon/pull/207/commits/2bcda3d1052a5da450071ec4301631a821e8b0ff#diff-904f54b080aa5edaf24e329308190d936166fff3aab8c0eaf9ff957b53dbf170L412